invenio_rdm_records/config.py

```diff
@@ -555,6 +557,17 @@ def make_doi(prefix, record):
    def lock_edit_published_files(service, identity, record=None, draft=None):
 """
 
+RRM_CONTENT_MODERATION_HANDLERS = {
```

Comment:
question: is the `RRM` intended or a typo?

Reply:
Nice catch, totally missed this one!